app/app.go

```diff
@@ -385,15 +396,37 @@ func New(
 	cdc := encodingConfig.Amino
 	interfaceRegistry := encodingConfig.InterfaceRegistry
 
+	var identity age.Identity
+	{
+		if cast.ToString(appOpts.Get("mode")) == "validator" {
```

Comment:
seems [mode](https://github.com/tendermint/tendermint/pull/6241/files#diff-003ce8db64b969b85ae335f805c56b16bed98a637ddfb4ac8da5ca04f6632ed9R102) config hasn't been include in main release, should we check seed_mode instead

Reply:
yeah, it seems only exists in pystarport now.

we do have a small issue that a normal node operator need to input a passphrase to unlock the keyring though.

Reply:
we'll need to require validator operator to set this config then.